### PR TITLE
feat(binary): Allow skip binary downloading

### PR DESCRIPTION
**Pull request by bangn**

### CODE REVIEW

File: standalone/install.spec.ts [hunk 1]
```diff
@@ -92,4 +92,20 @@ describe('Install', () => {
...
+
+		beforeEach(() => {
+			process.env.PACT_SKIP_BINARY_INSTALL = 'true';
+		});
+
+		afterEach(() => {
+			process.env = { ...OLD_ENV };
+		});
+
+		it.only('Should not download it', () => {
```
Comment (TimothyJones) [inline standalone/install.spec.ts:107]: I think this might be better as `it`, not `it.only`.

**Comment by TimothyJones** [comment]
Thanks for this!

Could you tell us a bit about the use case? Does the [download location setting](https://github.com/pact-foundation/pact-node#pact-download-location) cover it?

File: standalone/install.spec.ts (hunk 1 above)
Comment (bangn) [reply standalone/install.spec.ts]: My bad :face_palm:. Fixed 3efffc5c4bb08fe

**Comment by bangn** [comment]
> Thanks for this!
> 
> Could you tell us a bit about the use case? Does the [download location setting](https://github.com/pact-foundation/pact-node#pact-download-location) cover it?

Sorry, I should mention this is to fix issue #236

**Comment by bangn** [comment]
In addition, [download local setting](https://github.com/pact-foundation/pact-node#pact-download-location) only change the location of the binary.
This feature is to ignore the binary totally. The usecase I can think of (please
correct me if im wrong) is that the binary is already downloaded so user would
like not to download it again.

**Comment by mefellows** [comment]
Long time no see @bangn!

**Comment by TimothyJones** [comment]
Cool. That's a good rationale. We should add some documentation to the readme too before we release. Would you be able to do that in this PR?

**Comment by TimothyJones** [comment]
Also, sorry to nitpick - commits starting with `feat` or `fix` end up in the release notes - could you do a force push that changes the `feat` in `feat(binary): Remove only from the test` to something like `chore`?

**Comment by bangn** [comment]
> Long time no see @bangn!

Hi @mefellows, long time no see :D

@TimothyJones

> could you do a force push that changes the feat in feat(binary): Remove only from the test to something like chore?

Done!

> We should add some documentation to the readme too before we release. Would you be able to do that in this PR?

I'll do that. Thank you

**Comment by TimothyJones** [comment]
Releasing this in 10.11.0

**Pull request merged**